inspector.md

```diff
@@ -22,7 +22,19 @@ these widget trees. It can be helpful when:
 * Undestanding existing layouts
 * Diagnosing layout issues
 
-For a demonstration of the Inspector, please see the recent
+![IntelliJ Flutter Inspector Window](/images/intellij/visual-debugging.png)
+
+
+The typical workflow using the inspector is to click "Select widget" on the Flutter Inspector toolbar and then click on the device to select a widget. The selected widget will then be highlighted on the device and in the widget tree.
```

Comment:
Perhaps instead: "To start, click "Select widget" ...."

Reply:
done